### PR TITLE
Display clients and channels in a list instead of using cards

### PR DESCRIPTION
**Pull request by manuIO**
Signed-off-by: Manuel Imperiale <manuel.imperiale@gmail.com>

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -33,6 +33,7 @@ import { ChannelsStore } from './core/store/channels.store';
 import { ClientsStore } from './core/store/clients.store';
 import { UiStore } from './core/store/ui.store';
 import { AuthStore } from './core/store/auth.store';
+import { MatTableModule } from '@angular/material/table';
```
Comment (malidukica) [inline src/app/app.module.ts]: MatTableModule should go inside the MaterialModule, that's where the material modules should be imported, just not to pollute the app.module too much.

File: src/app/app.module.ts [hunk 2]
```diff
@@ -60,7 +61,8 @@ export function factory(authenticationService: AuthenticationService) {
     MaterialModule,
     FlexLayoutModule,
     ReactiveFormsModule,
-    MobxAngularModule
+    MobxAngularModule,
+    MatTableModule,
```
Comment (malidukica) [inline src/app/app.module.ts]: MatTableModule should go inside the MaterialModule, that's where the material modules should be imported, just not to pollute the app.module too much.

File: src/app/components/channels/channels.component.scss [hunk 3]
```diff
@@ -6,4 +6,29 @@
     position: fixed;
     bottom: 12px;
     right: 12px;
-}
\ No newline at end of file
+}
+
+/* List container */
+.channel-list-container {
+  display: flex;
```
Comment (malidukica) [inline src/app/components/channels/channels.component.scss]: Should use flex-layout directives

File: src/app/components/clients/clients.component.ts [hunk 4]
```diff
@@ -8,6 +8,7 @@ import { ConfirmationDialogComponent } from '../shared/confirmation-dialog/confi
 import { AddClientDialogComponent } from './add-client-dialog/add-client-dialog.component';
 import { ClientsStore } from '../../core/store/clients.store';
 import { ChannelsStore } from '../../core/store/channels.store';
+import { MatTableDataSource } from '@angular/material';
```
Comment (malidukica) [inline src/app/components/clients/clients.component.ts]: Is this used now?

Comment (manuIO) [reply src/app/components/clients/clients.component.ts]: No, I'll remove this

File: src/app/components/channels/channels.component.scss (hunk 3 above)
Comment (manuIO) [reply src/app/components/channels/channels.component.scss]: ok, I'll use FlexLayoutModule

**Comment by manuIO** [comment]
@malidukica it should be ok now!

File: src/app/core/material/material.module.ts [hunk 5]
```diff
@@ -41,7 +42,8 @@ import {
     MatCheckboxModule,
     MatSliderModule,
     MatSlideToggleModule,
-    MatChipsModule
+    MatChipsModule,
+    MatTableModule
```
Comment (malidukica) [inline src/app/core/material/material.module.ts]: You need to export the MatTableModule in order for it to be visible in the app.

**Comment by malidukica** [comment]
@manuIO I've checked the changes and I tested it in Chrome, seems to be fixed. Just remove the ChannelsCard and ClientCard rules from *.scss files and I think we're good to go!

**Comment by manuIO** [comment]
@malidukica @drasko it can be merged now!

**Pull request merged**